app/index.js

```diff
@@ -538,12 +539,19 @@ for (const button of buttons) {
 function showFace(flowback = false) {
     let skipQuestion = false;
 
-    // go through all views and set to none, show jsonFlow
+    // go through all views and set to none
     allViews.map((v) => {
         v.style.display = "none";
     });
-    jsonFlow.style.display = "inline";
-
+    
+    // check if numerical input is required and set Flow
+    if (covidFlow[currentView].type === "numerical") {
```

Comment:
@LionelKarlen I get an error on this line when I click `No` for  `Do you feel tired`. I think it's something to do with the flow.json not being finished, but yet going on to the last question

```
App: Unhandled exception: TypeError: Cannot read property 'type' of undefined
  ? at app/index.js:548,1
  ? at app/index.js:634,13
  ? at app/index.js:533,17
```

Reply:
@pjayathissa I also found this error while testing but because this also happens on master I didn't address it in this PR. Instead, I've opened an [issue](https://github.com/pjayathissa/cozie-covid/issues/17). I think it's best if we addressed this in a separate PR.